sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageQuerySource.java

```diff
@@ -48,6 +49,7 @@
       @Nullable String location,
       @Nullable String queryTempDataset,
       @Nullable String kmsKey,
+      DataFormat format,
```

Comment:
Should this be `@Nullable`? It looks like you pass null to this argument in `BigQueryIOStorageQueryTest`

Reply:
Yes, I think it should be nullable, adding the change

Reply:
Looks good, thanks